labs/lab-4/server.py

```diff
@@ -52,28 +52,28 @@
     # LAB 4 --------------------------------------------------
 
     # # write a query to insert 3 records into the Courses table
-    # query_1 = """ """
-    # db.session.execute(text(query_1))
-    # db.session.commit()
+    query_1 = """INSERT INTO Courses (CourseID, CourseName, Semester, Year) VALUES (120, 'Intro to Programming', 'Fall', 2020), (220, 'Software Development 1', 'Spring', 2021), (221, 'Software Development 2', 'Fall', 2021)"""
+    db.session.execute(text(query_1))
+    db.session.commit()
 
     # # write a query to select all records from the Courses table
-    # query_2 = """ """
-    # courses = db.session.execute(text(query_2))
-    # print(f"\n\n----------- Courses Table")
-    # for course in courses:
-    #     print(course)
+    query_2 = """SELECT * FROM Courses """
```

Comment:
The reason your queries are "struggling" (this made me laugh) is just because you're missing double quotes around table names and column names. For example, this should be 

```
SELECT * FROM "Courses";
```

And don't forget your semicolons!

Reply:
I think I fixed it!